PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewEmulatorForm.java

```diff
@@ -285,6 +286,7 @@ private void preEmulatorSetupCheck() {
 		sparkHistoryEndpoint = sparkHistoryEndpointField.getText().trim().replaceAll("/+$", "");
 		ambariEndpoint = ambariEndpointField.getText().trim().replaceAll("/+$", "");
 
+		AppInsightsClient.create(Messages.HDInsightCreateLocalEmulator, null);
```

Comment:
Fixed

---
In reply to: [120866533](https://github.com/Microsoft/azure-tools-for-java/pull/603#discussion_r120866533) [](ancestors = 120866533)